src/sentry/auth/access.py

```diff
@@ -259,7 +267,8 @@ def from_request(request, organization=None, scopes=None):
             projects=project_list,
             sso_is_valid=sso_is_valid,
             requires_sso=requires_sso,
-            open_access_policy=True,
+            open_access_policy=bool(organization.flags.allow_joinleave),
```

Comment:
maybe if we're going to add another flag we should just rename the existing flag to something like "has_global_access" and then I think it'd be more appropriate to combine the flags

my concern with yet-another flag is just theres a lot of branches things can go down to make a decision, and my concer w/ overloading "open_access_policy" is it becomes less obvious what it means

Reply:
👍 i like the idea of just combining into a more generic flag, will do that